docs/guides/usage/tracing.livemd

```diff
@@ -12,7 +12,25 @@ IO.puts("Installed matcha version: #{Application.spec(:matcha, :vsn)}")
 <!-- livebook:{"output":true} -->
 
 ```
-Installed matcha version: 0.1.4
+* Getting matcha (https://github.com/christhekeele/matcha.git - stable)
+remote: Enumerating objects: 2059, done.        
+remote: Counting objects: 100% (313/313), done.        
+remote: Compressing objects: 100% (227/227), done.        
+remote: Total 2059 (delta 108), reused 143 (delta 77), pack-reused 1746        
+Receiving objects: 100% (2059/2059), 486.56 KiB | 1.89 MiB/s, done.
+Resolving deltas: 100% (1200/1200), done.
+Resolving Hex dependencies...
+Resolution completed in 0.025s
+New:
+  recon 2.5.2
+* Getting recon (Hex package)
+==> recon
+Compiling 6 files (.erl)
+Generated recon app
+==> matcha
+Compiling 17 files (.ex)
+Generated matcha app
```

Comment:
I think this output is always going to change between runs, which is why I left the version status report below as the only truncated output.